deploy/addons/aliyun_mirror.json

```diff
@@ -40,7 +40,7 @@
   "k8s.gcr.io/cluster-autoscaler": "registry.cn-hangzhou.aliyuncs.com/google_containers/cluster-autoscaler",
   "k8s.gcr.io/coredns/coredns": "registry.cn-hangzhou.aliyuncs.com/google_containers/coredns",
   "k8s.gcr.io/defaultbackend": "registry.cn-hangzhou.aliyuncs.com/google_containers/defaultbackend",
-  "k8s.gcr.io/echoserver": "registry.cn-hangzhou.aliyuncs.com/google_containers/echoserver",
+  "polyverse/node-echo-server": "registry.cn-hangzhou.aliyuncs.com/google_containers/echoserver",
```

Comment:
I dont know who miantains "polyverse" I rather we use gcr images maintained by google or minikube team

Reply:
Here's a list of public GCR images: https://console.cloud.google.com/gcr/images/google-containers/GLOBAL